mipengine/controller/api/error_handlers.py

```diff
@@ -32,13 +33,15 @@ def handle_bad_user_input(error: BadUserInput):
 
 @error_handlers.app_errorhandler(InsufficientDataError)
 def handle_privacy_error(error: InsufficientDataError):
-    print(f"Insufficient Data Error: \n " + error.message)
+    ctrl_logger.getLogger(__name__).error(
```

Comment:
Not your fault but please make this ".info" . It's not a "problem" that this error occured.

Only in the unexpected exception, just like you have it there.